### PR TITLE
[JA] translation for gnatcatcher article

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Adds new translation of the gnatcatcher article in Japanese. 
Translated by Alconost

### CODE REVIEW

File: site/ja/docs/privacy-sandbox/gnatcatcher/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+_ネットワークアドレス変換_は、1 つの一意の IP アドレス でコンピューターのグループを表せるプロセスです。
+{% endAside %}
+
+ブラウザーは、Multiplexed Application Substrate over QUIC Encryption (MASQUE) を使用して、IP privatizing server (IPPS) 経由で HTTP トラフィックを転送します。 サーバーによって認識される HTTP トラフィックの IP アドレスは、ブラウザーの IP アドレス ではなく IPPS の IP アドレスです。 IPPS に対してトラフィックの内容を隠すために、ブラウザーはエンドツーエンドの暗号化を使用します。
+
+## Gnatcatcher の機能の仕組み
+
+Gnatcatcher は、Near-Path NAT と Willful IP Blindness の両方の使用を提案しています。 Near-Path NAT (または別のプロキシソリューション) は、ベースラインのデフォルトとして の宛先サーバーからの IP アドレスを使用します。
+
+悪用を防止するために追加の制御が必要な Web サービスプロバイダーのサブセットがあります。 これらのプロバイダーは、クライアントが直接接続できるように、Willful IP Blindness の準拠を証明するように選択できます。 その準拠は、監査と認証によってに保証される可能性があります。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/gnatcatcher/index.md:50]:
```suggestion
悪用を防止するために追加の制御が必要な Web サービスプロバイダーのサブセットがあります。 これらのプロバイダーは、クライアントが直接接続できるように、Willful IP Blindness の準拠を証明するように選択できます。 その準拠は、監査と認証（Certification）によってに保証される可能性があります。
```

File: site/ja/docs/privacy-sandbox/gnatcatcher/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Gnatcatcher'
+subhead: >
+  クロスサイトの隠されたトラッキングを防ぐため、個々のユーザーの IP アドレスを常に非表示にする。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/gnatcatcher/index.md:5]:
```suggestion
  クロスサイトの隠されたトラッキングを防ぐため、個々のユーザーの IP アドレスを常に隠します。
```

File: site/ja/docs/privacy-sandbox/gnatcatcher/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+description: >
+  クロスサイトの隠されたトラッキングを防ぐため、個々のユーザーの IP アドレスを常に非表示にする。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/gnatcatcher/index.md:7]:
```suggestion
  クロスサイトの隠されたトラッキングを防ぐため、個々のユーザーの IP アドレスを常に隠します。
```

File: site/ja/docs/privacy-sandbox/gnatcatcher/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+*  この提案はどのブラウザーにも実装されていません。
+*  [プライバシーサンドボックスタイムライン](https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline) は、Gnatcatcher およびその他のプライバシーサンドボックス 提案の実装タイミングを示します。
+
+## この提案が必要な理由
+
+IP アドレスは、クライアントに一意の識別子を提供するために作成されます。 これにより、トラフィックをインターネット経由でルーティングできます。 IP アドレスは 一定期間にわたって安定しているため、ファーストパーティ全体でユーザーを識別できます。
+
+Gnatcatcher、つまりグローバルネットワークアドレス変換と Audited および Trusted CDN または HTTP-Proxy Elimination Reidentification の組み合わせは、IP Blindness に対して提案された ソリューションです。 この提案は、[Willful IP Blindness](https://github.com/bslassey/ip-blindness/blob/master/willful_ip_blindness.md) と [Near-Path NAT](https://github.com/bslassey/ip-blindness/blob/master/near_path_nat.md) (またはプロキシ関連ソリューション) (Willful IP Blindness に参加していない接続の場合) という他の 2 つの提案と組み合わせることを提言しています。
+
+つまり、デフォルトでは IP アドレスは非表示になります。 サイトは、IP アドレスを悪用せずに直接接続を使用していること を証明できる場合があります。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/gnatcatcher/index.md:27]:
```suggestion
つまり、デフォルトでは IP アドレスは隠されます。 サイトは、IP アドレスを悪用せずに直接接続を使用していること を証明できる場合があります。
```

**Comment by ilyaspiridonov** [comment]
> A few changes required.

Thank you, I committed the changes!

**Review by agektmr — APPROVED** [review]
LGTM

**Review by devnook — APPROVED** [review]

**Pull request merged**